runtime/plugins/console/Dockerfile

```diff
@@ -0,0 +1,30 @@
+FROM alpine:3.14
```

Comment:
I was confused by the `plugins` word -> the `console` layer is a normal layer (unless it changed here?), I don't think we should have a concept of plugins, that's one more concept to keep track off? We might as well have this layer in `runtime/` directly?

Reply:
The problem with the `console` layer is that it's not 1 layer per PHP version and it's also not one layer per CPU architecture. But in any case this was an old idea that ended up adding more complexity than necessary. I deleted the entire folder.